web/src/auth/LoginPage.js

```diff
@@ -15,14 +15,16 @@
 import React from "react";
 import {Link} from "react-router-dom";
 import {Button, Checkbox, Col, Form, Input, Result, Row, Spin} from "antd";
+import {Radio, Divider} from 'antd';
 import {LockOutlined, UserOutlined} from "@ant-design/icons";
+import * as UserBackend from "../backend/UserBackend";
 import * as AuthBackend from "./AuthBackend";
 import * as ApplicationBackend from "../backend/ApplicationBackend";
 import * as Provider from "./Provider";
 import * as Util from "./Util";
 import * as Setting from "../Setting";
 import SelfLoginButton from "./SelfLoginButton";
-import {GithubLoginButton, GoogleLoginButton} from "react-social-login-buttons";
+import { GithubLoginButton, GoogleLoginButton } from "react-social-login-buttons";
```

Comment:
Don't change code formatting here, if you want, you can propose a new PR for this